components/brave_vpn/resources/panel/components/select-region-list/index.tsx

```diff
@@ -60,19 +62,26 @@ function ConnectButton(props: ConnectButtonProps) {
 interface RegionCityProps {
   cityLabel: string
   serverInfo: string
+  serverCount: number
   selected: boolean
   connectionButton: React.ReactElement
 }
 
 function RegionCity(props: RegionCityProps) {
+  const { result: serverInfo } = usePromise(
+    async () => PluralStringProxyImpl.getInstance().getPluralString(
+      'braveVpnServerSelectionCityInfo',
+      props.serverCount),
+    [props.serverCount])
+
   return (
     <S.RegionCity selected={props.selected}>
       <S.RegionCityInfo>
         <S.RegionCityLabel selected={props.selected}>
           {props.cityLabel}
         </S.RegionCityLabel>
         <S.CityServerInfo selected={props.selected}>
-          {props.serverInfo}
+          {serverInfo}
```

Comment:
@fallaciousreasoning testing `PluralStringProxyImpl` like this but it doesn't work. `serverInfo` seems `undefined`.
something I missed to use it?

Reply:
Interesting - does it work at all if you try and use it from the console (i.e. `window['PSPI'] = PluralStringProxyImpl` - might help narrowing down whats going wrong.

I wonder if its because `braveVpnServerSelectionCityInfo` is defined as a string as well as a PluralString?

Reply:
I think this behavior comes from `UntrustedWebUIController`.
UntrustedWebUI doesn't have webui binding for `chrome.send()`
As`PluralStringHandler` is based on `chrome.send()`, I think vpn ui can't get message even it registered that handler.

As an example, `RewardsPanelUI` is based on `TopChromeWebUIController`
and it enables webui bindings explicitely.

I assume that vpn panel handles external data so it's based on untrusted.
Not sure we could change vpn panel to `TopChromeWebUIController`.
How about going with first try?

Reply:
Aaaah okay, happy to leave this then. Sorry about that @simonhong

Reply:
yup, deleted second commit.